### ISSUE TITLE
Error message not correct when no files are found

### ISSUE THREAD

**Issue by nvnieuwk** [askimed/nf-test#86]
Hi, I'm trying to run tests with tags but am getting this error:
```bash
$ nf-test test --tag snapshots

🚀 nf-test 0.7.3
https://code.askimed.com/nf-test
(c) 2021 - 2023 Lukas Forer and Sebastian Schoenherr

Found 1 files in test directory.
Error: No tests or test directories containing scripts that end with *.test provided.
```

While the directory looks like:
```bash
tests/
├── annotate_test.test
├── default.test
├── fails.test
├── filter_test.test
├── full_gvcf.test
├── full_wes.test
├── full_wgs.test
├── gemini_test.test
├── inputs
│   ├── samplesheet.csv
│   ├── samplesheet_gvcf.csv
│   └── test.ped
├── nextflow.config
└── snapshot.test
```

It does work when I renamed the `snapshot.test` file to `snapshot.nf.test`. Can you maybe change this check to actually look for `*.test` files instead `*.nf.test`?

-Nicolas

**Comment by lukfor** [comment]
Hi Nicolas, good point, we will fix it so that the error message is on par with the expected behaviour.

The main idea to use `*.nf.test` as an extension was to make clear that this file is a test of a nextflow file.

Happy to discuss this with you guys! What's your take? What's your motivation to use `*.test` instead of `*.nf.test`?

**Comment by nvnieuwk** [comment]
For me it's actually all the same :), Just wanted to point out the error message was a bit confusing. I'll add the `.nf` to all files

**Comment by lukfor** [comment]
Fixed in 0.9.2-rc2.
